### ISSUE TITLE
[Accessibility] Bibliography list style does not change in html format

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-dev/quarto-cli#1841]
Regardless of whether users provide csl file or not in yaml, all the references at the bottom of the document only display each reference URL. Is this the expected behavior? How can users change the reference list style in html format?

**Comment by dragonstyle** [comment]
I'm not able to reproduce this - the test document I'm using appears to be properly responsive to the provided CSL (I've attached the simple test document and files as a zip below). Do you have a simple test case I could use to reproduce the issue?

[CSL Test Example](https://github.com/quarto-dev/quarto-cli/files/9316476/test.zip)

It is possible that it is somehow related to the specific CSL style - I've included a couple of samples in the example that vary the style significantly, but perhaps there is another issue specific to a particular style...

**Comment by jooyoungseo** [comment]
It seems like the visual output is correctly presented with all the authors information according to any provided CSL; however, such information is semantically-hidden to screen readers.

**Comment by jooyoungseo** [comment]
This is because of `doc-bibliography` role. I don't see associated aria-label there. However, some screen readers (NVDA and Narrator on Windows) recognize this role. JAWS does not. Still need to test with VoiceOver on Mac. JAWS does not support this role; however, this is the issue of JAWS. I will further investigate this issue.

**Comment by dragonstyle** [comment]
Ok thank you- I better understand the issue you’re seeing now! If you find there is something that we or pandoc should be doing let us know…

**Comment by jooyoungseo** [comment]
I am coming back to this issue. `doc-biblioentry` role is deprecated so I need to do some tweaks. @jjallaire -- Could you please let me know where I can find the html base template to address this issue?

**Comment by jjallaire** [comment]
@jooyoungseo The very best thing you can do is make a contribution to the core Pandoc html template which is maintained here: https://github.com/jgm/pandoc/blob/master/data/templates/default.html5. That way all Pandoc users will benefit from the change.

**Comment by jooyoungseo** [comment]
This issue has been reported via [jgm/pandoc#8354](https://github.com/jgm/pandoc/issues/8354) and [the latest commit](https://github.com/jgm/pandoc/commit/b05c380f6ff4375cbc27955653fd31310ff2f94d) has been pushed. Will be addressed with the next pandoc release.

**Comment by jjallaire** [comment]
Great! Thank you :-) I'll close this issue here was we'll automatically get the update when the new version of pandoc lands.
